registry/virtual.go

```diff
@@ -62,8 +467,15 @@ func OverwriteAppName(virtualSpaceName, appSlug, newName string) error {
 	overwrite["name"] = newName
 
 	id := getAppID(appSlug)
-	_, err = db.Put(context.Background(), id, overwrite)
-	return err
+	if _, err = db.Put(context.Background(), id, overwrite); err != nil {
+		return err
+	}
+
+	if err := regenerateOverwrittenTarballs(virtualSpaceName, appSlug); err != nil {
```

Comment:
We can just `return regenerateOverwrittenTarballs(virtualSpaceName, appSlug)` to avoid a few lines

Reply:
:up: